requirements.txt

```diff
@@ -18,6 +18,7 @@ idna<2.9,>=2.5
 requests==2.25.1
 packaging==21.3
 python-dotenv==0.21.0
+setuptools<65.6
```

Comment:
can we add a comment to remember why we did this and remove it when fixed plz ?

Reply:
Right 👍